### DISCUSSION TITLE
How to indicate a condition that a MARC field/subfield does NOT exist?

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#334] [Q&A] [answered]
Sometimes, we will want to create a condition that a certain field does not exist in MARC. For example, "has related work of work" is really only a desirable mapping from 700**$0 if there is no $i present. Same with "related agent of work" and other broad properties...we really only want them if there is no relationship designator. How do we want to express these kinds of things in our mapping? I have an idea, where we just use "!" as the value for a condition to indicate that the subfield is not found in the data. We could also say something like "absent" or "not exists" ... but we should probably be consistent about it, no? Has anyone else run into a negative condition like this?

**Comment by gerontakos** [comment, marked as answer]
XPath has a function not(). We could use that; it's much like your exclamation mark.
So:
Condition:
not(700$i)

??

**Reply by CECSpecialistI** [reply]
I like that, it's more human-readable.
